ssh2/agent.c

```diff
@@ -1,18 +1,28 @@
-/* Generated by Cython 0.29.21 */
+/* Generated by Cython 0.21.1 */
```

Comment:
Generating Cython files with `0.21.1` *will not work*. Cython needs to be updated on your system.